wallets/metamask/src/pages/HomePage/actions/renameAccount.ts

```diff
@@ -0,0 +1,22 @@
+import type { Page } from '@playwright/test'
+import Selectors from '../selectors'
+import { z } from 'zod'
+
+const Account = z.object({
+  name: z.string().refine((value) => {
+    return value.trim().length > 0 && !/^Account\s*\d+/.test(value);
```

Comment:
We cannot verify this using the word `Account` because MetaMask can be used in different languages. That's why our code needs to be language-agnostic.

Even though I love this, I'm not sure what's the best way to handle this here. If you don't have any other idea, feel free to just drop this schema validation and add a `TODO` message:
`// TODO: See if we can add language-agnostic schema validation here.`